### PR TITLE
last bit of styling for today

### PR DESCRIPTION
**Pull request by gorakong**
@VALE46GP @JayBo7 @jcelder

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ class App extends React.Component {
     const { reviews, rating, aggregateRating } = this.state;
     return (
       <div className={classNames({ [styles.body]: true, container: true })}>
```
Comment (jcelder) [inline client/src/components/App.jsx:76]: @gorakong Why are you using the classnames package?

Comment (gorakong) [reply client/src/components/App.jsx:76]: @jcelder  I used it to allow for the usage of multiple classes (i.e., for bootstrap classes & my own css)

Comment (jcelder) [reply client/src/components/App.jsx:76]: @gorakong I see. I don't believe you need to do that with CSS modules. You can import your classes into an object and then refer to them as keys on that object. For example if you wanted to add classes from bootstrap and classes from your own css you could potentially do the following.
```
import styles from './styles.css'
import bootstrap from '/path/to/bootstrap.css'

<div
  className={[styles.coolStyle, bootstrap.btn].join(' ')}
>
</div>
```

I am going to approve this PR for now, but I would strongly recommend refactoring your class names to use CSS modules.

Comment (gorakong) [reply client/src/components/App.jsx:76]: Sounds good! I will be making this refactor.

**Pull request merged**